optimum/intel/openvino/configuration.py

```diff
@@ -914,6 +910,13 @@ def post_init(self):
         if self.gptq and self.lora_correction:
             raise ValueError("The GPTQ and LoRA Correction algorithms can't be applied simultaneously")
 
+        valid_group_size_fallback_values = [e.value for e in nncf.GroupSizeFallbackMode]
```

Comment:
wondering if we should add a check somewhere (like in https://github.com/huggingface/optimum-intel/blob/75d6b7d3bc9544487e2111a610b59f8d62e0ef89/optimum/intel/openvino/__init__.py#L46) to ensure that the nncf version is compatible with our integration (in our case >= v2.19 I imagine ?)

Reply:
thanks!